src/App.scss

```diff
@@ -3,18 +3,81 @@ html {
 }
 
 .App {
+  width: 1200px;
+  margin: 0 auto;
+
   &__header {
+    position: relative;
     display: flex;
     align-items: center;
     gap: 20px;
+    justify-content: center;
+
+    color: white;
+    background-color: rgb(245, 14, 14);
+    border-radius: 20px;
+  }
+
+  &__clear {
+    position: absolute;
+    right: 45px;
+  }
+
+  &__button {
+    width: 100px;
+    padding: 10px 15px;
+
+    font-weight: bold;
+    font-size: 16px;
+
+    background-color: #fff000;
+    border: none;
+    border-radius: 12px;
+
+    cursor: pointer;
+
+    transition: 200ms;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties.